tests/NuGet.IndexingTests/CamelCaseFilterTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Lucene.Net.Analysis.Standard;
+using Lucene.Net.Analysis.Tokenattributes;
+using NuGet.Indexing;
+using Xunit;
+using Lucene.Net.Util;
+using NuGet.IndexingTests.TestSupport;
+
+namespace NuGet.IndexingTests
+{
+    public class CamelCaseFilterTests
+    {
+        [Theory, MemberData("TheoryData")]
+        public void Theory(string text, TokenAttributes[] expected)
```

Comment:
Name a test so it is meaningful.

e.g. TokenizingReturnsTheExpectedTermAndOffsets